src/main/java/uk/gov/justice/digital/delius/data/api/ManagedOffenderCrn.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ManagedOffenderCrn {
+    private String offenderCrn;
+
```

Comment:
We made the decision to return a sparse response for a few reasons e.g. to avoid the need for LAO processing.  The idea being you could use the identifiers to then fetch any remaining information you need, and rely on the existing behaviour of those endpoints.

Technically it'd be easy to add the offender details in to this response, however our architect @mxco86 is back on Monday so it might be best to wait till then before making any changes - if that's okay?